### ISSUE TITLE
Color contrast on focus indicator is not high enough

### ISSUE THREAD

**Issue by bbsmooth** [freeCodeCamp/chapter#2044]
### Describe the bug:
The light blue focus indicator does not have enough color contrast with the white background to be accessible. Using the color picker tool in gimp, the light blue color used for the focus indicator is approximately `#8ec2ed`. This only has a contrast ratio of `1.89:1` with the white background. It needs to be at least `3:1`.

Additionally, when it is around a blue button the contrast between the dark blue button and the focus indicator is only `2.12:1`. And with the red buttons it is only slightly better at `2.17:1`. 

### Solution

The focus ring definitely needs to have a contrast of at least `3:1` with the white background. As for contrast with the darker buttons, one option would be to place a little distance between the button and the focus indicator so that the white background separates them (this can be as little as 1px). As long as you have that separation then you don't need to worry about the contrast between the focus indicator and the button color since they aren't touching. For example, if you were using `outline` for the focus indicator then you could use `outline-offset: 1px` to create that separation.

**Comment by Sboonny** [comment]
Went through the pages, didn't find a button that need outline so I am closing this
